src/custom-elements/docs/docs.json

```
@@ -1612,6 +1612,153 @@
         ]
       }
     },
+    {
+      "filePath": "src/components/dropdown/dropdown.tsx",
+      "encapsulation": "scoped",
+      "tag": "chi-dropdown",
+      "docs": "",
+      "docsTags": [],
+      "usage": {},
+      "props": [
+        {
+          "name": "active",
+          "type": "boolean",
+          "mutable": false,
+          "attr": "active",
+          "reflectToAttr": false,
+          "docs": "to set the state of Dropdown.",
```

Capitalise all descriptions.

```suggestion
          "docs": "To set the state of Dropdown.",
```